tests/testthat/reporters/check.txt

```diff
@@ -14,38 +14,46 @@ FALSE isn't true.
 
 ── 5. Error: Error:1 (@tests.R#33)  ────────────────────────────────────────────
 stop
-1: stop("stop") at reporters/tests.R:33
+Backtrace:
+ 1. NULL
```

Comment:
I'm surprised to see a `NULL` in a backtrace

Reply:
Fixed by using `seq2()` instead of `:` https://github.com/r-lib/rlang/commit/a334055b